vignettes/articles/sliding.Rmd

```diff
@@ -288,78 +287,151 @@ available in epipredict.
 
 ## Version-aware and unaware forecasting
 
-### Example using case data from US states 
+### Example using case data from US states
 
 We will now employ a forecaster that uses properly-versioned data (that would
-have been available in real-time) to forecast future COVID-19 case rates from
-current and past COVID-19 case rates for all states. That is, we can make
-forecasts on the archive, `x`, and compare those to forecasts on the latest
-data, `x_latest` using the same general set-up as above. For version-aware
-forecasting, note that `x` is fed into `epix_slide()`, while for version-unaware
-forecasting, `x_latest` is fed into `epi_slide()`. 
-
-```{r make-ar-kweek-asof}
-k_week_version_aware <- function(ahead = 7, version_aware = TRUE) {
-  if (version_aware) {
-    epix_slide(
-      x,
-      ~ arx_forecaster(
-        .x, "case_rate", c("case_rate", "percent_cli"),
-        args_list = arx_args_list(ahead = ahead)
-      ) %>%
-        extract2("predictions"),
-      before = 120 - 1,
-      ref_time_values = fc_time_values,
-      new_col_name = "fc"
+have been available in real-time) to forecast the 7 day average of future
+COVID-19 case rates from current and past COVID-19 case rates and death rates
+for all states. That is, we can make forecasts on the archive, `x`, and compare
+those to forecasts on the latest data, `x_latest` using the same general set-up
+as above. For version-aware forecasting, note that `x` is fed into
+`epix_slide()`, while for version-unaware forecasting, `x_latest` is fed into
+`epi_slide()`. Note that in this example, we use a geo-pooled approach (using
+combined data from all US states and territories) to train our model.
+
+```{r load-data}
```

Comment:
I suspect most of this should be run offline (rather than everytime the vignette builds). Can we save the archive elsewhere?

Reply:
@rnayebi21 I think you still need to address this one.

Reply:
Oh sure sorry. Do you recommend saving it in its own file and then just uploading the whole file to github?